hackathon_site/event/jinja2/event/signup.html

```diff
@@ -0,0 +1,43 @@
+{% extends "event/form_base.html" %}
+
+{% block form_title %}Sign up{% endblock %}
+{% block form_button %}Sign up{% endblock %}
+
+{% block form_img %}<img src="{{ static('event/images/landing.svg') }}" />{% endblock %}
+               
+{% block form_inputs %}
+    <div class="input-field col s12">
+        <input id="first_name" type="text" class="validate">
+        <label for="first_name">first name</label>
+    </div>
+    <div class="input-field col s12">
+        <input id="last_name" type="text" class="validate">
+        <label for="last_name">last name</label>
+    </div>
+    <div class="input-field col s12">
+        <input id="email" type="email" class="validate">
+        <label for="email">email</label>
+    </div>
+    <div class="input-field col s12">
+        <input id="password" type="password" class="validate">
+        <label for="password">password</label>
+        <span class="helper-text">Must contain blah blah</span>
+    </div>
+    <div class="input-field col s12">
+        <input id="password" type="password" class="validate">
```

Comment:
Two things with the same id. `password-confirm` or something here (I think the Django form will set the id down the line anyway)